Objects/longobject.c

```diff
@@ -181,7 +181,67 @@ _PyLong_FromNbInt(PyObject *integral)
         Py_DECREF(result);
         return NULL;
     }
-    return (PyLongObject *)result;
+    return result;
+}
+
+
+/* XXX Should be replaces with PyNumber_Index after the end of the
```

Comment:
Can you please write a short description of the function and write more explicitly that it's deprecated?

Something like:


Convert the given object to a PyLongObject using the nb_int slot or nb_index, if available. Raise TypeError if either
nb_int and nb_index slot are not available or the result of the call to nb_int or nb_index returns something not of type int.

Reply:
It is documented in the header file. But well, I'll duplicate the description here.